app/javascript/components/widgets/manifest.js

```diff
@@ -31,7 +31,7 @@ import * as treeCoverLocated from './widgets/land-cover/tree-cover-located';
 import * as emissions from './widgets/climate/emissions';
 import * as woodyBiomass from './widgets/climate/whrc-biomass/';
 import * as soilBiomass from './widgets/climate/soil-organic';
-import * as emissionsDeforestation from './widgets/climate/emissions-deforestation';
+// import * as emissionsDeforestation from './widgets/climate/emissions-deforestation';
```

Comment:
should we remove this comments?

Reply:
for now no. This is enabled in another branch.